tests/chainer_tests/functions_tests/loss_tests/test_cross_covariance.py

```diff
@@ -11,14 +11,14 @@
 from chainer.testing import attr
 
 
-def _cross_covariance(y, z):
+def _cross_covariance(y, z, dtype=numpy.float32):
```

Comment:
Please remove default value because anyone don't use it.

Reply:
Removed!